Src/FluentAssertions/Equivalency/Comparands.cs

```diff
@@ -43,18 +43,8 @@ public Type CompileTimeType
     /// <summary>
     /// Gets the run-time type of the current expectation object.
     /// </summary>
-    public Type RuntimeType
-    {
-        get
-        {
-            if (Expectation is not null)
-            {
-                return Expectation.GetType();
-            }
-
-            return CompileTimeType;
-        }
-    }
+    public Type RuntimeType =>
```

Comment:
🔧 We need to disable this rule in Quodana. I don't think we should _always_ do this.

Reply:
That rule should have been disabled in #2229

Reply:
This should be resolved, right?